tests/test_klaviyo_start_date.py

```diff
@@ -132,9 +131,15 @@ def run_start_date(self, expected_streams, days = 0):
                     for bookmark_key_value in bookmark_key_sync_2:
                         self.assertGreaterEqual(bookmark_key_value, start_date_2_epoch)
 
-                    # Verify the number of records replicated in sync 1 is greater than the number
-                    # of records replicated in sync 2 for stream
-                    self.assertGreater(record_count_sync_1, record_count_sync_2)
+                    if stream in ['failed_to_deliver_automated_response', 'failed_to_deliver', 'subscribed_to_email']:
+                        # Verify the number of records replicated in sync 1 is greater than or equal to
+                        # the number of records replicated in sync 2 for mentioned streams as
+                        # the data is available for only 1 day for these streams
+                        self.assertGreaterEqual(record_count_sync_1, record_count_sync_2)
```

Comment:
Since data is available for only 1 day, the count of records will be same for any start date. Start date value does not include time precision.
Hence, we need to keep this consition.